akka-actor/src/main/scala/akka/pattern/CircuitBreaker.scala

```diff
@@ -106,7 +106,7 @@ class CircuitBreaker(
   scheduler:                Scheduler,
   maxFailures:              Int,
   callTimeout:              FiniteDuration,
-  resetTimeout:             FiniteDuration,
+  val resetTimeout:         FiniteDuration,
```

Comment:
I wonder if this is binary compatible when the class is not final, could break if a subclass has a `def resetTimeout` (not a likely scenario, and it's not supposed to be subclassed)